requirements.txt

```diff
@@ -0,0 +1,9 @@
+ansunit==0.1.7
```

Comment:
No, I don't want to fix the direct dependencies. I want to lock all dependencies but be flexible with the direct dependencies. There should be a separate lockfile.